addons/l10n_ke/models/res_company.py

```diff
@@ -0,0 +1,17 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    l10n_ke_oscu_is_active = fields.Boolean(
+        string="Is OSCU active?",
+        help="Whether this company is set up for OSCU flows.",
+        compute='_compute_l10n_ke_oscu_is_active',
+    )
+
+    def _compute_l10n_ke_oscu_is_active(self):
+        self.l10n_ke_oscu_is_active = False
```

Comment:
Maybe add some comment here explaining that it's overridden in another module in enterprise